React/03_class/shopping_cart/src/components/Products/action-creator.js

```diff
@@ -20,6 +20,30 @@ export const setProducts = () => async (dispatch) => {
   }
 };
 
+export const sortProducts = (option) => async (dispatch) => {
+  try {
+    if(option === '0'){
+      dispatch({
+        type: 'SORT_BY_INCREASING_ORDER'
```

Comment:
Should be SORT_BY_INCREASING_PRICE instead of ORDER? We only know that's what it's sorting by if we refer to the function it's calling.